### DISCUSSION TITLE
L'app. di Home Assistant non invia più i dati dopo l'aggiornamento ad Android 12

### DISCUSSION THREAD

**Discussion by StefanoVitale69** [indomus/forum#1336] [HUB: Home Assistant]
Salve, qualche giorno fa ho aggiornato il mio Samsung Note 20 ad Android 12 ed improvvisamente l'app. Home Assistant non invia più i dati dei sensori del cellulare. Infatti, se accedo ad Home Assistant, l'ultimo aggiornamento ad es. della posizione del cellulare è fermo a 5 gg. fa. 
Non ho più notato gli aggiornamenti automatici che prima faceva quando ad es. il Wi-Fi del cellulare si connetteva al router di casa o dell'ufficio.
Qualcuno sa da cosa dipende e come posso risolvere?
Grazie

**Comment by Scialla** [comment]
Ho avuto un problema analogo su tutti i telefoni collegati; ho disinstallato l'app companion e l'ho reinstallata risolvendo.

Ogni tanto mi si inchioda ancora su una posizione diversa da quella in cui sono (solo l'app companion), cmincio a pensare a interferenze a casa.... troppi giocattoli forse :D
